robyn/__init__.py

```diff
@@ -173,6 +173,16 @@ def start(self, url: str = "127.0.0.1", port: int = 8080):
         logger.info(f"Robyn version: {__version__}")
         logger.info(f"Starting server at {url}:{port}")
 
+        endpoint = {}
+
+        routes = self.router.get_routes()
```

Comment:
I think we should write the logs in the add_route method, instead of doing the get_routes here - it's minor, but it slows down the startup time